examples/systemd/conf/agent/default.conf

```diff
@@ -18,6 +21,12 @@ plugins {
         }
     }
 
+    # NodeAttestor "tpm" {
+    #     plugin_cmd = "/usr/bin/tpm_attestor_agent"
+    #     plugin_checksum = ""
+    #     plugin_data {}
+    # }
```

Comment:
is this required? can you add a comment?

Reply:
Just as an example. I can remove it.